Jenkinsfile

```diff
@@ -48,6 +48,10 @@ install_tabular_all = """
     python3 -m pip install --upgrade -e tabular/[all,tests]
 """
 
+install_multimodal = """
+    python3 -m pip install --upgrade -e multimodal/[tests]
+"""
```

Comment:
Must add multimodal to this file:

https://github.com/awslabs/autogluon/blob/master/autogluon/setup.py

Reply:
Added.